RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -118,28 +189,31 @@ public bool Generate(ProcessedArgs args)
                 createdJarFilePath = BuildPlugin(definition, args.OutputDirectory);
             }
 
-            LogSummary(createdJarFilePath, generatedSqaleFile, licenseAcceptancePackages);
+            LogSummary(createdJarFilePath, generatedSqaleFile);
 
             return createdJarFilePath != null;
         }
 
-        private void LogSummary(string createdJarFilePath, string generatedSqaleFile, IEnumerable<IPackage> licenseAcceptancePackages)
+        private void LogSummary(string createdJarFilePath, string generatedSqaleFile)
         {
             if (generatedSqaleFile != null)
             {
-                // Log a messsage about the generated sqale file at the end of the process: if we
+                // Log a message about the generated sqale file at the end of the process: if we
```

Comment:
What did you decide in relation to the sqale file generation - that it's ok to generate sqale files for dependencies, and that it's ok that the message about the file being created is output at the end of the overall run?